### PR TITLE
readme: fix installation link

### PR DESCRIPTION
**Pull request by khaneliman**
Resolves https://github.com/nix-community/nixvim/issues/2142

### CODE REVIEW

**Comment by MattSturgeon** [comment]
This won't work, see https://github.com/nix-community/nixvim/issues/1625 and https://github.com/nix-community/nixvim/pull/2038

**Comment by MattSturgeon** [comment]
Ah, my mistake. I skimmed over that this is modifying the link in the README, not the relative links in the user-guide.

File: README.md [hunk 1]
```diff
@@ -89,7 +89,7 @@ If you have any question, please use the [discussions page](https://github.com/n
 >
 > If you want to use NixVim with nixpkgs 24.05 you should use the `nixos-24.05` branch.
 
-For more detail, see the [Installation](https://nix-community.github.io/nixvim/install.html#installation) section of our documentation.
+For more detail, see the [Installation](https://nix-community.github.io/nixvim/user-guide/install.html) section of our documentation.
```
Comment (MattSturgeon) [inline README.md:92]:
```suggestion
For more detail, see the [Installation](https://nix-community.github.io/nixvim) section of our documentation.
```

I believe this is the correct link.

Comment (khaneliman) [reply README.md]: Sounds good, I was thinking about that change after reading the linked issues.

**Review by MattSturgeon — APPROVED** [review]
Awesome, thanks!

**Comment by khaneliman** [comment]
@mergifyio queue

**Pull request merged**